tests/utils.go

```diff
@@ -0,0 +1,10 @@
+package tests
+
+import "testing"
+
+func CheckErr(t *testing.T, err error) {
```

Comment:
Going to start to put test utils and other things (like the mocks above) in this `tests` package for reuse.

Reply:
You can consider the pkg used in `integration_test.go` for assertions.
It's very short, and when fails it automatically provides nice output messages.
For this case would be: `require.Nil(t, err)` (in the test itself, so no need for `CheckErr` func).
I use it for other things like: `require.Equal(t, 10, someNumber)` and have many helping things that shrkings `if` lines.

Reply:
That's great, will switch to that.